test/kitchen/drivers/vagrant-driver.yml

```diff
@@ -1,12 +1,11 @@
 ---
```

Comment:
Lines 12 and 13 of this file needs to be changed, it should be:
```
    ENV['TEST_PLATFORMS'] ||= "centos7,roboxes/centos7|centos8,roboxes/centos8"
    test_platforms = ENV['TEST_PLATFORMS'].split('|').map { |p| p.split(',') }
```
to match the other drivers, as `genconfig` uses the same `TEST_PLATFORMS` format for all drivers.